src/Components/Ignitor/src/Ignitor.csproj

```diff
@@ -1,10 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <!-- Intentionally targeting netcoreapp3.0 because we do not want to update the Benchmark client to use 3.1 -->
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
     <IsShippingPackage>false</IsShippingPackage>
-    <DefineConstants>$(DefineConstants);IGNITOR</DefineConstants>
+    <DefineConstants>TRACE;IGNITOR</DefineConstants>
```

Comment:
Rarely a good idea to completely overwrite `$(DefineConstants)`. Why isn't this _extending_ the constants anymore❔ Also curious why `TRACE` would be helpful if enabled in a benchmark app.

Reply:
Yeah oops that was an artifact of debugging, whoops.

Reply:
Excellent -- an "oops" _and_ a "whoops" 😃